### PR TITLE
Convert bootstraps/enhanced/membership to ES6

### PR DESCRIPTION
**Pull request by svillafe**
## What does this change?

This PR transforms `/bootstraps/enhanced/membership.js` to ES6 standard. 

## Tested in CODE?
It should be tested in CODE.

### CODE REVIEW

File: static/src/javascripts-legacy/projects/common/modules/identity/account-profile.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ define([
                     var tabs = self.accountProfileForms.querySelector(self.classes.tabs);
 
                     require.ensure([], function (require) {
-                        require('bootstraps/enhanced/membership').init();
+                        require('../javascripts/bootstraps/enhanced/membership').init();
```
Comment (SiAdcock) [inline static/src/javascripts-legacy/projects/common/modules/identity/account-profile.js]: Why is this necessary?

**Review by SiAdcock — APPROVED** [review]

**Comment by svillafe** [comment]
Tested in CODE and it is working.

**Comment by svillafe** [comment]
It is working on PROD.
